### PR TITLE
ENT-6101 drop unsupported platforms

### PR DESCRIPTION
**Pull request by Lex-2008**
And add Ubuntu 20

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thanks for submitting a pull request! Maybe @craigcomstock can review this?

**Review by craigcomstock — APPROVED** [review]

File: build-scripts/package [hunk 1]
```diff
@@ -249,7 +249,7 @@ case "$PACKAGING" in
     (cd "$BASEDIR/$PKG/pkg"; export DEB_BUILD_OPTIONS="$DEB_BUILD_OPTIONS"; dpkg-buildpackage -b -us -uc -rfakeroot) || false
 
     # Create TAR package
-    TARBALL="$BASEDIR/$PKG/pkg/$PKG-$DEB_VERSION$safe_prefix.$OS$os_version_major.$ARCH.pkg.tar.gz"
+    TARBALL="$BASEDIR/$PKG/pkg/$PKG-$DEB_VERSION$safe_prefix.$os_version_major.$ARCH.pkg.tar.gz"
```
Comment (vpodzime) [inline build-scripts/package:252]: Shouldn't `$os_version_major` be dropped as well?

**Review by vpodzime — CHANGES_REQUESTED** [review]
The first commit message of the first commit is wrong. It should use `<` instead of `<=` and also, the commit leaves Ubuntu 16 in the list(s) so clearly `Ubuntu < 16` is unsupported.

**Review by vpodzime — APPROVED** [review]
Looks good to me now.

**Pull request merged**